hapi-utils/src/main/java/com/hedera/services/keys/Ed25519Utils.java

```diff
@@ -15,17 +15,8 @@
  */
 package com.hedera.services.keys;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.OutputStreamWriter;
-import java.security.DrbgParameters;
-import java.security.KeyPair;
-import java.security.NoSuchAlgorithmException;
-import java.security.Provider;
-import java.security.SecureRandom;
+import java.io.*;
```

Comment:
Seems something is different on your machine. When I run spotless, it replaces wildcards with the imports .. 🤔